src/styles/mixins/layout.scss

```diff
@@ -70,12 +70,14 @@
 
 /**
  * Establish a vertical (margin) rhythm for this element's immediate
- * children (i.e. put equal space between children). Currently set to 1em.
+ * children (i.e. put equal space between children).
+ *
+ * @param $size [var.$layout-margin--medium]: Spacing size (padding)
  * @FIXME Find workaround for SvgIcon exception
  */
-@mixin vertical-rhythm {
+@mixin vertical-rhythm($size: var.$layout-margin) {
```

Comment:
Here I am adding a parameter to `vertical-rhythm` because I encountered the first need for it to be another size...

Reply:
👍